javascript/ql/src/Security/CWE-830/FunctionalityFromUntrustedSource.ql

```diff
@@ -0,0 +1,169 @@
+/**
+ * @name Inclusion of untrusted functionality by a HTML element.
+ * @description Including untrusted functionality by a HTML element
+ *              opens up for potential man-in-the-middle attacks.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 8.1
```

Comment:
I think we can adjust the severity a bit down.  
We don't have any CVEs tagged with `cwe-830`, so we don't have any data to go on.  
But I think `6.0` could be appropriate. That also somewhat fits with it being a `warning` query. 

```suggestion
 * @security-severity 6.0
```

Reply:
👍